Dockerfile

```diff
@@ -1,72 +1,50 @@
 FROM python:3.8-buster as python-base
-
 LABEL maintainer="TACC-ACI-WMA <wma_prtl@tacc.utexas.edu>"
-
 ARG DEBIAN_FRONTEND=noninteractive
-
-ENV PYTHONUNBUFFERED 1
-
-# https://python-poetry.org/docs/configuration/#using-environment-variables
-ENV POETRY_VERSION=1.4.0 \
-    POETRY_HOME="/opt/poetry" \
-    POETRY_VIRTUALENVS_IN_PROJECT=true \
-    POETRY_NO_INTERACTION=1 \
-    PYSETUP_PATH="/opt/pysetup" \
-    VENV_PATH="/opt/pysetup/.venv"
```

Comment:
If I am grokking correctly, because line 11 (in the changed column -->) is now using the container's default Python, there is no need to define the `PYSETUP_PATH`. The `POETRY_HOME` and `VENV_PATH` are no longer being used in the `PATH` definition either because the default locations would now be correct. So all three values can get dropped.